source/main.bmx

```diff
@@ -2397,6 +2397,48 @@ Type TSaveGame Extends TGameState
 
 	Global _nilNode:TNode = New TNode._parent
 	Function RepairData(savegameVersion:Int, savegameConverter:TSavegameConverter = null)
+		If savegameVersion < 21
```

Comment:
Nur zur Sicherheit: die TemplateCollections durchzugehen reicht aus? Es gibt nicht noch irgendwelche schon erzeugten Einträge (upcoming news etc.), in denen sich noch Variablen verstecken könnten.

Reply:
Wenn aus Templates eine News generiert wird, sollte sie keine Expressions mehr enthalten (in meinen Tests war dies auch so).